### PR TITLE
Feature/add loading view

### PR DESCRIPTION
**Pull request by ellSamuel**
Add loading view (@bocato 's dev sprint)

### CODE REVIEW

File: solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+struct LoadingView: View {
+    var message: String?
```
Comment (bocato) [inline solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift]: **[CLARIFICATION]**
Do you need this to be a `var`?
```suggestion
    let message: String?
```

File: solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    
+    var body: some View {
+        VStack {
+            ProgressView()
```
Comment (bocato) [inline solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift:9]: Also, the view is supposed to center itself on the screen... We are missing this here.

File: solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        }
+    }
+}
+
+#if DEBUG
+struct LoadingView_Previews: PreviewProvider {
+    static var textExample: String? = "Loading..."
+    static var previews: some View {
+        LoadingView(message: textExample)
+    }
```
Comment (bocato) [inline solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift]: **[SUGGESTION]**
Consider also checking how it behaves in Dark mode...
```suggestion
    static let textExample: String = "Loading..."
    static var previews: some View {
        LoadingView(message: textExample)
            .preferredColorScheme(.light)
        LoadingView(message: textExample)
            .preferredColorScheme(.dark)
    }
```

**Review by bocato — CHANGES_REQUESTED** [review]

File: solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift (hunk 3 above)
Comment (ellSamuel) [reply solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift]: Yea, that's clever

**Pull request closed without merging**